pkg/kratos/service.go

```diff
@@ -55,12 +55,12 @@ type UiErrorMessages struct {
 }
 
 func (s *Service) CheckSession(ctx context.Context, cookies []*http.Cookie) (*kClient.Session, []*http.Cookie, error) {
-	ctx, span := s.tracer.Start(ctx, "kratos.FrontendApi.ToSession")
+	ctx, span := s.tracer.Start(ctx, "kratos.Service.ToSession")
 	defer span.End()
 
 	session, resp, err := s.kratos.FrontendApi().
 		ToSession(ctx).
-		Cookie(cookiesToString(cookies)).
+		Cookie(httpHelpers.CookiesToString(cookies)).
```

Comment:
question: does this `CookiesToString` function take care also of other attributes for the cookies like path, domain, expires etc.., or is it on the sdk to take care of that?

Reply:
It doesn't - it just gets the cookies names and values (see [here](https://github.com/canonical/identity-platform-login-ui/blob/3965a2f2c05aa49d0f8d446190ad07a8904fc70c/internal/misc/http/helpers.go#L17), the rest is on sdk side